content/applications/productivity/discuss/overview/get_started.rst

```diff
@@ -1,71 +1,78 @@
+:guilabel:`Marked as unread`.
+
+.. image:: get_started/reactions-discuss.png
    :align: center
    :alt: View of an inbox message and its action options in Odoo Discuss
 
-The messages tagged as *Mark as Todo* are also shown in *Starred*, while the ones *Marked as Read*
-are moved to *History*.
+Clicking :guilabel:`Mark as Todo` on a message causes it to appear on the :guilabel:`Starred` page,
+while clicking :guilabel:`Mark as Read` moves the message to :guilabel:`History`.
 
 .. image:: get_started/starred_messages.png
    :align: center
    :alt: View of messages marked as todo in Odoo Discuss
 
-Start Chatting
+Start chatting
 ==============
 
-The first time you log in to your account, OdooBot sends you a message asking for permission to
-receive desktop notifications to chats. If accepted, you receive push notifications to the messages
-sent to you despite of where you are in Odoo.
+The first time a user logs in to their account, OdooBot sends a message asking for permission to
+send desktop notifications for chats. If accepted, the user will receive push notifications on their
+desktop for the messages they receive, regardless of where the user is in Odoo.
 
 .. image:: get_started/odoobot_push.png
    :align: center
    :alt: View of the messages under the messaging menu emphasizing the request for push
          notifications for Odoo Discuss
 
 .. tip::
-   To stop receiving desktop notifications, reset the notifications settings of your browser.
+   To stop receiving desktop notifications, reset the notifications settings of the browser.
 
-To start a chat, click on *New Message* on the *Messaging Menu*, or go to *Discuss* and send a
-*Direct Message*.
+To start a chat, go to the :menuselection:`Discuss` app and click on the :guilabel:`+` icon next to
+:guilabel:`Direct Messages` or :guilabel:`Channels` in the left menu of the dashboard.
 
 .. image:: get_started/channels_direct_messages.png
    :align: center
    :height: 400
-   :alt: View of discuss’s panel emphasizing the titles channels and direct messages in Odoo Discuss
+   :alt: View of Discuss’s panel emphasizing the titles channels and direct messages in Odoo Discuss
 
-You can also create :doc:`public and private channels <team_communication>`.
+A company can also easily create :doc:`public and private channels <team_communication>`.
 
-Mentions in the chat and on the Chatter
+Mentions in the chat and on the chatter
 ---------------------------------------
 
-| To mention a user within a chat or the chatter type *@user-name*; to refer to a channel, type
-  *#channel-name*.
-| A notification is sent to the user mentioned either to his *Inbox* or through email, depending
-  on his settings.
+To mention a user within a chat or the chatter, type *@user-name*; to refer to a channel, type
+*#channel-name*. The user mentioned will be notified in their *Inbox* or through an email, depending
+on their communication settings.
```

Comment:
User input should use literal formatting not italics, and _inbox_ should be lowercase since the text is not in a guilabel
```suggestion
To mention a user within a chat or the chatter, type `@user-name`; to refer to a channel, type
`#channel-name`. The user mentioned will be notified in their *inbox* or through an email, depending
on their communication settings.
```